CHANGELOG.md

```diff
@@ -1,5 +1,49 @@
 # Changelog for Elixir v1.9
 
+## Releases
```

Comment:
Maybe include this section or a summary of it? https://github.com/elixir-lang/elixir/blob/master/lib/mix/lib/mix/tasks/release.ex#L48-L81

Reply:
grwat idea, done!